ui/app/components/clients/page/token.hbs

```diff
@@ -3,199 +3,99 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<div class="box is-sideless is-fullwidth is-marginless is-bottomless">
-  <p class="has-bottom-margin-xl">
-    This dashboard will surface Vault client usage over time. Clients represent a user or service that has authenticated to
-    Vault.
-    <Hds::Link::Inline @href={{doc-link "/vault/docs/concepts/client-count"}}>Documentation is available here</Hds::Link::Inline>.
-    Date queries are sent in UTC.
-  </p>
-  <h2 class="title is-6 has-bottom-margin-xs">
-    {{this.versionText.label}}
-  </h2>
-  <div data-test-start-date-editor class="is-flex-align-baseline">
-    {{#if this.formattedStartDate}}
-      <p class="is-size-6" data-test-date-display>{{this.formattedStartDate}}</p>
-      <Hds::Button
-        class="has-left-margin-xs"
-        @text="Edit"
-        @color="tertiary"
-        @icon="edit"
-        @iconPosition="trailing"
-        {{on "click" (fn (mut this.showBillingStartModal) true)}}
+{{#if this.byMonthActivityData}}
```

Comment:
After thinking about this more, I think we should handle this how we did before and instead of showing this for a single month: 
<img width="1078" alt="Screenshot 2024-01-25 at 9 51 56 AM" src="https://github.com/hashicorp/vault/assets/68122737/2462d898-8bda-4abc-924b-7c31465f5fba">

We show the stat boxes. 
<img width="1052" alt="Screenshot 2024-01-25 at 9 52 38 AM" src="https://github.com/hashicorp/vault/assets/68122737/a57ad729-f9e2-45b1-97da-a54591b74661">
<img width="1043" alt="Screenshot 2024-01-25 at 9 53 35 AM" src="https://github.com/hashicorp/vault/assets/68122737/eef54584-efcb-42ea-b632-49f39003b73d">

Reply:
Sure. So no charts if a single month is queried then? I think when we last discussed this each component was sort of handling the same condition with it's own empty state but I can just adjust the condition in the token page component to render the `UsageStats` component if there is no monthly activity data or if the query is for a single month.

Reply:
Yeah, I think that works. Essentially I'd render the chart if `isDateRange` is true, and then the different stats depending on whether it's the current month or not

Reply:
Since the new vs. total breakdown is available on the overview I have updated to render the `UsageStats` component when a single month is queried (both current or historical).